src/API/Util/Currencies.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace OnPay\API\Util;
+
+class Currencies {
+
+    const ALPHA3_CURRENCIES = array(
+        'AUD' => array('exponent'=>2, 'iso4217' => 36),
+        'CAD' => array('exponent'=>2, 'iso4217' => 124),
+        'CNY' => array('exponent'=>2, 'iso4217' => 156),
+        'CZK' => array('exponent'=>2, 'iso4217' => 203),
+        'DKK' => array('exponent'=>2, 'iso4217' => 208),
+        'ISK' => array('exponent'=>0, 'iso4217' => 352),
+        'INR' => array('exponent'=>2, 'iso4217' => 356),
+        'JPY' => array('exponent'=>0, 'iso4217' => 392),
+        'NZD' => array('exponent'=>2, 'iso4217' => 554),
+        'NOK' => array('exponent'=>2, 'iso4217' => 578),
+        'RUB' => array('exponent'=>2, 'iso4217' => 643),
+        'SGD' => array('exponent'=>2, 'iso4217' => 702),
+        'ZAR' => array('exponent'=>2, 'iso4217' => 710),
+        'SZL' => array('exponent'=>2, 'iso4217' => 748),
+        'SEK' => array('exponent'=>2, 'iso4217' => 752),
+        'CHF' => array('exponent'=>2, 'iso4217' => 756),
+        'ECP' => array('exponent'=>2, 'iso4217' => 818),
+        'GBP' => array('exponent'=>2, 'iso4217' => 826),
+        'USD' => array('exponent'=>2, 'iso4217' => 840),
+        'EUR' => array('exponent'=>2, 'iso4217' => 978),
+        'UAH' => array('exponent'=>2, 'iso4217' => 980),
+        'PLN' => array('exponent'=>2, 'iso4217' => 985),
+        'BRL' => array('exponent'=>2, 'iso4217' => 986),
+    );
+
+    const ISO4217_CURRENCIES = array(
```

Comment:
I am not a big fan of having the same static data 2 times, is it possible to solve this in a different way to ensure an update is never "halfly" done ?